src/Symfony/Component/Security/Http/Attribute/IsGranted.php

```diff
@@ -24,15 +28,15 @@
 final class IsGranted
 {
     /**
-     * @param string|Expression            $attribute     The attribute that will be checked against a given authentication token and optional subject
-     * @param array|string|Expression|null $subject       An optional subject - e.g. the current object being voted on
-     * @param string|null                  $message       A custom message when access is not granted
-     * @param int|null                     $statusCode    If set, will throw HttpKernel's HttpException with the given $statusCode; if null, Security\Core's AccessDeniedException will be used
-     * @param int|null                     $exceptionCode If set, will add the exception code to thrown exception
+     * @param string|Expression|(\Closure(TokenInterface $token, mixed $subject, AccessDecisionManagerInterface $accessDecisionManager, AuthenticationTrustResolverInterface $trustResolver): bool) $attribute     the attribute that will be checked against a given authentication token and optional subject, or a callable that will be called to determine access
+     * @param array|string|Expression|(\Closure(array<string, mixed>, Request): mixed)|null                                                                                                         $subject       An optional subject - e.g. the current object being voted on
+     * @param string|null                                                                                                                                                                           $message       A custom message when access is not granted
+     * @param int|null                                                                                                                                                                              $statusCode    If set, will throw HttpKernel's HttpException with the given $statusCode; if null, Security\Core's AccessDeniedException will be used
+     * @param int|null                                                                                                                                                                              $exceptionCode If set, will add the exception code to thrown exception
```

Comment:
I'm not sure aligning makes sense here

Reply:
Did so to please fabbot but I'm fine here to not align with the first line, it's a bit too much